changelog/v1.13.20/add-gwp-hpa-v2-support.yaml

```
@@ -0,0 +1,5 @@
+changelog:
+  - type: NEW_FEATURE
```

```suggestion
  - type: HELM
```

https://github.com/solo-io/go-utils/tree/main/changelogutils#changelog-files This will show up in our public docs with other helm values, and also ensure that changelog-bot (which is currently failing) passes

---

Done 🙏🏼